test/slugify.js

```diff
@@ -276,4 +276,12 @@ describe('slugify', () => {
   it('should preserve leading/trailing replacement characters if option set', function () {
     t.equal(slugify(' foo bar baz ', { trim: false }), '-foo-bar-baz-')
   })
+
+  it('should correctly handle empty strings in charmaps', () => {
+    slugify.extend({'a': ''})
+    t.equal(slugify('cat'), 'ct');
+
+    delete require.cache[require.resolve('../')]
+    slugify = require('../')
```

Comment:
Not relevant to this PR, but `slug` has a `.reset()` method to make this more ergonomic. Maybe `slugify` could do something similar.